plugin/core/windows.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, sublime: Any, settings: Settings) -> None:
         self._settings = settings
 
     def for_window(self, window: WindowLike, workspace: ProjectFolders,
-                   configs: ConfigRegistry) -> DocumentHandler:
+                   configs: WindowConfigManager) -> DocumentHandler:
         return WindowDocumentHandler(self._sublime, self._settings, window, workspace, configs)
```

Comment:
Wrong type here.

Reply:
The DocumentHandler is a Protocol. The WindowDocumentHandler fits that Protocol.